webapp/src/ts/services/integration-api.service.ts

```diff
@@ -0,0 +1,54 @@
+import { Injectable } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+
+import { LanguageService } from '@mm-services/language.service';
+import { Select2SearchService } from '@mm-services/select2-search.service';
+import { MRDTService } from '@mm-services/mrdt.service';
+import { MarkdownService } from '@mm-services/markdown.service';
+import { SettingsService } from '@mm-services/settings.service';
+import { AndroidApiService } from '@mm-services/android-api.service';
+import { DbService } from '@mm-services/db.service';
+// todo migrate enketo
```

Comment:
Similar to my question above about the `simprints` TODO, is this to be handled in a separate PR?

Reply:
As above.

Reply:
Actually, my bad 👍🏻 it's not as above. 

I'm intentionally splitting the Contact summary + Integration api + Enketo into different PRs to make it more manageable. 
So, it will be handled into a future PR.